src/tests/test_integration/test_movies.py

```
@@ -0,0 +1,538 @@
+        .offset(offset)
+        .limit(per_page)
+    )
+    result = await db_session.execute(stmt)
+    expected_movies = result.scalars().all()
+
+    expected_movie_ids = [movie.id for movie in expected_movies]
+    returned_movie_ids = [movie["id"] for movie in response_data["movies"]]
+
+    assert expected_movie_ids == returned_movie_ids, "Movies on the page mismatch."
+
+    expected_prev_page = f"/theater/movies/?page={page - 1}&per_page={per_page}" if page > 1 else None
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)